README.md

```diff
@@ -2,4 +2,4 @@
 
 The web application source files are in app/.  This is an application using the Laravel PHP framework.  See its readme for details.
 
-Some tools for gathering location information from other websites have been added to importers/.  These include downloading and CSV generator tools for OpenStreetMap, toiletmap.org and tools for converting CSV into appropriate changes to our seed data files in the Laravel application.
+Some tools for gathering location information from other websites have been added to importers/.  These include downloading and CSV generator tools for OpenStreetMap, toiletmap.org and tools for converting CSV into appropriate changes to our seed data files in the Laravel application.
```

Comment:
Remove the commit that did this.  I think it had the message, haoyixian1994@hotmail.com.

The reason to remove that commit is that it is not related to the issue you're trying to fix and wouldn't help anyone using 'git blame' on this file.